### PR TITLE
fixing #18 by making producers write scan info in the protobuf as the…

### PR DESCRIPTION
**Pull request by northdpole**
… api points out

### CODE REVIEW

File: producers/producer.go [hunk 1]
```diff
@@ -67,8 +67,9 @@ func WriteDraconOut(
 		cleanIssues = append(cleanIssues, iss)
 		log.Printf("found issue: %+v\n", iss)
 	}
-
-	return putil.WriteResults(toolName, cleanIssues, outFile)
+	scanStartTime := os.Getenv("EnvDraconStartTime")
```
Comment (VJftw) [inline producers/producer.go:70]: should be: `DRACON_SCAN_TIME`

File: producers/producer.go [hunk 2]
```diff
@@ -67,8 +67,9 @@ func WriteDraconOut(
...
+	scanUUUID := os.Getenv("EnvDraconScanID")
```
Comment (VJftw) [inline producers/producer.go:71]: should be: `DRACON_SCAN_ID`

**Comment by northdpole** [comment]
> Then, you'll also need to get dracon to make this available to producers by copying https://github.com/thought-machine/dracon/blob/master/resources/patches/patch-dracon-enriched.Consumer.yaml#L24 to https://github.com/thought-machine/dracon/blob/master/resources/patches/patch-dracon.Producer.yaml which will template the DRACON_SCAN_ID, ... pipeline params into producer tasks :)

done :)

**Comment by northdpole** [comment]
bump for review

**Comment by VJftw** [comment]
I'm going to merge this, but will create an issue to fix the proto compilation (remove from plz and use the `scripts/gen-proto.sh` so we can compile without asking everyone to install protoc and libs which can be quite frustrating

**Pull request merged**